root/defaults/nginx/authelia-server.conf.sample

```diff
@@ -1,28 +1,52 @@
-## Version 2022/08/22 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/authelia-server.conf.sample
+## Version 2022/09/22 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/authelia-server.conf.sample
 # Make sure that your authelia container is in the same user defined bridge network and is named authelia
 
 location ^~ /authelia {
-
-    set $upstream_app authelia;
-    set $upstream_port 9091;
-    set $upstream_proto http;
-    proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+    include /config/nginx/proxy.conf;
+    include /config/nginx/resolver.conf;
+    set $upstream_authelia authelia;
+    proxy_pass http://$upstream_authelia:9091;
 }
 
 location = /authelia/api/verify {
     internal;
-
-    set $upstream_app authelia;
-    set $upstream_port 9091;
-    set $upstream_proto http;
-    proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+    if ($request_uri ~ [^a-zA-Z0-9_+-=\!@$%&*?~.:#'\;\(\)\[\]]) {
```

Comment:
can you remove this bit that's no longer needed?

Reply:
Done in https://github.com/linuxserver/docker-swag/pull/266/commits/b346b4610cdd43191a31dd4879b1f35768679d9c